.github/workflows/selfdrive_tests.yaml

```diff
@@ -61,13 +61,20 @@ jobs:
         cd $STRIPPED_DIR
         ${{ env.RUN }} "release/check-dirty.sh && \
                         MAX_EXAMPLES=5 $PYTEST -m 'not slow' selfdrive/car"
-    - name: static analysis
+    - name: Getting modified files
```

Comment:
CI should always run on all files, like pre-commit used to do

Reply:
Wouldn't that break the new goal of having the same pre-commit result in CI and locally?